crates/matrix-sdk/src/account.rs

```diff
@@ -623,4 +628,34 @@ impl Account {
         });
         Ok(self.client.send(request, None).await?)
     }
+
+    /// Set the given account data event.
+    pub async fn set_account_data<T>(
+        &self,
+        content: T,
+    ) -> Result<set_global_account_data::v3::Response>
+    where
+        T: GlobalAccountDataEventContent,
+    {
+        let own_user =
+            self.client.user_id().ok_or_else(|| Error::from(HttpError::AuthenticationRequired))?;
+
+        let request = set_global_account_data::v3::Request::new(&content, own_user)?;
+
+        Ok(self.client.send(request, None).await?)
+    }
+
+    /// Set the given raw account data event.
```

Comment:
Same  here, a bit more verbosity could help and an example wouldn't hurt either.

Reply:
Heh, I was kinda expecting this PR not to go through w/o more docs. Maybe I should have put it up as draft only 😅

Reply:
Ah right, you didn't request review. Sorry :sweat_smile:

Reply:
I did, actually. Personally I wouldn't need more docs and it was kinda late yesterday, I'll do better next time!

Reply:
Yeah, obviously I don't need the docs either, but the docs aren't for us in the first place.